extensions/objects/templates/svg-filename-editor.component.html

```diff
@@ -2,9 +2,10 @@
   <div ng-if="!$ctrl.isDiagramSaved()" class="diagram-editor-size-toolbar">
     <div class="input-box">Width: <input type="number" ng-model="$ctrl.diagramWidth" ng-blur="$ctrl.onWidthInputBlur()"></div>
     <div class="input-box">Height: <input type="number" ng-model="$ctrl.diagramHeight" ng-blur="$ctrl.onHeightInputBlur()"></div>
-    <i class="material-icons" title="<[$ctrl.getDiagramSizeInfo()]>">
-      &#xE887;
-    </i>
+    <div class="dimension-note">
+      This diagram has a maximum dimension of 490 X 550 px to ensure that it fits in the card and note that the change will
```

Comment:
Hm, seems a bit vague. What is "the card" and how does the user know when "SVG is created" -- maybe mention what action in the UI will trigger this?

Reply:
I have dropped the reason because the users need not know the technical reasons for the constraint. And I have added the UI trigger. Done.